app/views/layouts/application.html.slim

```diff
@@ -39,6 +39,9 @@ html
         = yield
   footer
     .aside-footer
+      p
+        - unless Git::BRANCH.empty?
+          '#{Git::BRANCH}@#{Git::COMMIT}
```

Comment:
Actually, on a second thought, maybe you could rename this module to `Version`, and the method could be `self.to_s`. What do you think ?